crates/code_generation/src/run.rs

```diff
@@ -147,6 +199,8 @@ fn generate_cycler_constructors(cyclers: &Cyclers) -> TokenStream {
                 communication_server.get_parameters_reader(),
                 #own_producer_identifier
                 #(#other_cycler_inputs,)*
+                recording_sender.clone(),
+                cycler_instances_to_be_recorded.contains(#cycler_instance_name),
```

Comment:
Please extract that variable to an earlier position like e.g. `enable_recording`

Reply:
Extracted to variable